baseclass/test/java/com/google/apigee/calloutsbaseclass/CalloutsBaseTest.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.apigee.calloutsbaseclass;
```

Comment:
use the same package as the tested class.  here you use callout s baseclass instead of callout baseclass.  Note the extra 's' here.